Test/dafny1/SchorrWaite.dfy

```diff
@@ -227,7 +227,7 @@ class Main {
       invariant forall k :: 0 < k < |stackNodes| ==>
                   stackNodes[k].children[stackNodes[k].childrenVisited] == stackNodes[k-1]
       // the original values of m.children[m.childrenVisited] for m's on the stack:
-      invariant forall k :: 0 <= k && k+1 < |stackNodes| ==>
+      invariant forall k {:matchinglooprewrite false} :: 0 <= k && k+1 < |stackNodes| ==>
```

Comment:
A change was made to SchorrWaite.dfy in master so it verifies faster, so maybe this change is no longer necessary.

Reply:
I fixed up `SchorrWaite.dfy` by using the loop invariants from `SchorrWaite-stages.dfy` (in the order given throughout the refinement modules in that file).